### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by iWeaker4you**
I have added the support to the Spanish language, here I have some screenshots so that it is appreciated in detail.

Captura de pantalla_2021-02-23_04-48-21
Captura de pantalla_2021-02-23_04-35-45
Captura de pantalla_2021-02-23_04-36-09

### CODE REVIEW

**Comment by flavioislima** [comment]
@iWeaker4you thanks a lot man, I was really missing the Spanish translations.

File: public/locales/es/translation.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    "quit": {
+      "message": "Hay operaciones pendientes, ¿estás seguro?",
+      "title": "Salir"
+    },
+    "runexe": {
+      "title": "Seleccione EXE para ejecutar"
+    },
+    "select": "Seleccionar",
+    "stopInstall": {
+      "keepInstalling": "",
```
Comment (flavioislima) [inline public/locales/es/translation.json:27]: It think there are some missing translations on this file like this one. Can you check? 
Other than that looks fine 👍🏽

File: src/components/Login.tsx [hunk 2]
```diff
@@ -174,7 +174,16 @@ export default function Login({ refresh }: Props) {
...
+              Türkçe 🇹🇷 -{' '}
+            </span>
+            <span
+              className={cx({
+                ['selectedLanguage']: currentLanguage === 'es',
+                ['language']: currentLanguage !== 'es',
+              })}
+              onClick={() => handleChangeLanguage('es')}
+            >
+              Español 🇲🇽
```
Comment (flavioislima) [inline src/components/Login.tsx:186]: I know this is a difficult question, but maybe we should use the Spanish flag instead? 
Even though then we should change the Portuguese one to the Portuguese flag 🤔

File: public/locales/es/translation.json (hunk 1 above)
Comment (iWeaker4you) [reply public/locales/es/translation.json:27]: @flavioislima I don't know what goes there, in all languages ​​it is empty including English :0
I have already changed the flags, the one for Spanish and Portuguese.

Comment (flavioislima) [reply public/locales/es/translation.json:27]: Hmm, that's weird. I need to check then because maybe this is missing somewhere.

Comment (flavioislima) [reply public/locales/es/translation.json:27]: You're right, those keys were duplicated for some reason. I'll fix soon.

**Pull request merged**